automathor.py

```
@@ -0,0 +1,202 @@
+
+    def parse_line(self, line: str) -> AnnotatedLine:
+        """Parse a line of git blame output."""
+        pattern = r"(\w+)\s+(\S+)\s+\(<([^>]+)>\s+(\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} [+-]\d{4})\s+(\d+)\)\s*(.*)"
+        match = re.match(pattern, line)
+        pattern_two = r"(\w+)\s+\(<([^>]+)>\s+(\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} [+-]\d{4})\s+(\d+)\)\s*(.*)"
+        match_two = re.match(pattern_two, line)
+        if match:
+            commit, filename, author_email, date_str, line_number, source_line = match.groups()
+            date_time = datetime.strptime(date_str, "%Y-%m-%d %H:%M:%S %z")
+            return AnnotatedLine(commit, filename, author_email, date_time, int(line_number), source_line)
+        elif match_two:
```

Ruff (0.8.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)